frontend/web/src/app/components/digital-part-list/digital-part-list.component.ts

```diff
@@ -37,73 +37,73 @@ export class DigitalPartListComponent implements OnInit, AfterViewInit {
   loadAndPopulate() {
     /* get users */
     this.digitalPartMockService.getDigitalParts().subscribe(
-      customers => {
+      (customers) => {
         this.digitalParts = customers;
 
         this.populate();
         this.prepareTriggers();
-      }, error => {
+      }, (error) => {
         alert(error.verbose_message);
-      }
+      },
     );
   }
 
   private populate() {
-    let _self = this;
+    const _self = this;
     /*manually generate columns*/
-    let columns = [{
-      title: "Namn",
-      field: "name",
-      sortable: true
+    const columns = [{
+      title: 'Namn',
+      field: 'name',
+      sortable: true,
     }, {
-      title: "Kund ID",
-      field: "id",
+      title: 'Kund ID',
+      field: 'id',
       visible: false,
-      sortable: true
+      sortable: true,
     }, {
       field: 'operate',
       title: '<span class="glyphicon glyphicon-cog">',
       align: 'center',
       events: {
-        'click .edit': function (e, value, row, index) {
+        'click .edit': function(e, value, row, index) {
           _self.router.navigate([_self.router.url, row.id]);
         },
-        'click .delete': function (e, value, row, index) {
-          _self.delete(row.id)
-        }
+        'click .delete': function(e, value, row, index) {
+          _self.delete(row.id);
+        },
       },
-      formatter: this.operateFormatter
+      formatter: this.operateFormatter,
     }];
 
-    let data = [];
+    const data = [];
     this.digitalParts.forEach((digitalPart) => {
       data.push({
         name: digitalPart.name,
         id: digitalPart.id,
       });
     });
 
-    (<any>this.table).bootstrapTable('destroy');
-    (<any>this.table).bootstrapTable(
+    (this.table as any).bootstrapTable('destroy');
+    (this.table as any).bootstrapTable(
       {
-        "data": data,
-        "columns": columns,
-        "sortStable": true,
-        "sortName": "name"
-      }
+        data: data,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Expected property shorthand in object literal ('{data}'). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3498)